acceptance/tests/sync/sync_catalog_test.go

```diff
@@ -86,15 +86,10 @@ func TestSyncCatalog(t *testing.T) {
 // The test will create a test service and a pod and will
 // wait for the service to be synced *to* consul.
 func TestSyncCatalogWithIngress(t *testing.T) {
-	t.Skip("TODO(fails): NET-8594")
-
 	cfg := suite.Config()
 	if cfg.EnableCNI {
 		t.Skipf("skipping because -enable-cni is set and sync catalog is already tested with regular tproxy")
 	}
-	if !cfg.UseEKS {
-		t.Skipf("skipping because -use-eks is not set and the ingress test only runs on EKS")
-	}
```

Comment:
Is this not true anymore? 🤔

Reply:
Yeah, this test isn't specific to EKS, as it works fine on Kind as well. I think originally the feature was built for folks who wanted to use it on EKS, but nothing about the feature acts on specific EKS annotations or relies on EKS itself. It doesn't even need an ingress controller deployed for the test itself, since the catalog sync only watches Ingress resources themselves.